src/sds_data_model/_dataframe.py

```diff
@@ -245,7 +249,7 @@ def _create_dummy_dataset(
     return (
```

Comment:
`mypy` is complaining because this bracketed expression returns `Any` instead of `None`. To fix this, just don't return anything
  
```suggestion
    (
```

Reply:
Addressed in https://github.com/Defra-Data-Science-Centre-of-Excellence/sds-data-model/pull/160/commits/7a423f9f5a671fa00256013d1781ad26797761f0